roles/serviceassurance/tasks/component_alertmanager.yml

```diff
@@ -0,0 +1,60 @@
+- name: Set default Alertmanager manifest
+  set_fact:
+    alertmanager_manifest: |
+      apiVersion: monitoring.coreos.com/v1
+      kind: Alertmanager
+      metadata:
+        labels:
+          alertmanager: '{{ meta.name }}'
+        name: '{{ meta.name }}'
+        namespace: '{{ meta.namespace }}'
+      spec:
+        replicas: 1
+        serviceAccountName: prometheus-k8s
+        serviceMonitorSelector:
+          matchLabels:
+            app: smart-gateway
+        storage:
+          class: {{ alertmanager_storage_class }}
+          resources: {{ alertmanager_storage_resources }}
+          selector: {{ alertmanager_storage_selector }}
+          volumeClaimTemplate:
+            spec:
+              resources:
+                requests:
+                  storage: {{ alertmanager_pvc_storage_request }}
+  when: alertmanager_manifest is not defined
+
+- name: Set default Alertmanager configuration manifest
+  set_fact:
+    alertmanager_config_manifest: |
+      apiVersion: v1
+      kind: Secret
+      metadata:
+        name: alertmanager-'{{ meta.name }}'
```

Comment:
Sorry small nit. Can you just drop the `alertmanager` from this `name` line? When creating an Alertmanager the operator already tacks on `-alertmanager` to the value here, resulting in `alertmanager-saf-default-alertmanager`.

Reply:
This is a secret, without alertmanager in the name, it'll become saf-default, I tested this again.

Reply:
```
oc describe secrets saf-default
Name:         saf-default
Namespace:    sa-telemetry
Labels:       <none>
Annotations:  <none>

Type:  Opaque

Data
====
alertmanager.yaml:  163 bytes
```

Reply:
Oops misread. Sorry. Thanks!